zerver/lib/markdown/__init__.py

```diff
@@ -135,7 +135,15 @@ def normal_compile(pattern: str) -> Any:
 
 @one_time
 def get_compiled_stream_link_regex() -> Pattern:
-    return verbose_compile(STREAM_LINK_REGEX)
+    # Not using verbose_compile as it adds ^(.*?) and
+    # (.*?)$ which cause extra overhead of matching
+    # pattern which is not required.
+    # With new InlineProcessor these extra patterns
+    # are not required.
+    return re.compile(
+        f"{STREAM_LINK_REGEX}",
```

Comment:
`f"{STREAM_LINK_REGEX}"` is just `STREAM_LINK_REGEX`.

Reply:
Yeah I made these changes.